tests/auto/desktop-qt5/downloadmgr/tst_downloadmgr.qml

```diff
@@ -40,7 +40,7 @@ Item {
         focus: true
         active: true
         anchors.fill: parent
-        onViewInitialized: {
+        Component.onCompleted: {
             webViewport.addMessageListener("embed:downloadpicker")
             appWindow.mozViewInitialized = true
```

Comment:
addMessageListener, loadFrameScript, and e.g. setPreference can be called before view initialized and internals take care of handling them. That said, "appWindow.mozViewInitialized = true" should be onViewInitialized.